guides/micronaut-graphql-todo/src/main/resources/db/migration/V1__schema.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE author -- <1>
```

Comment:
should make the capitalization consistent - it looks weird that the only all caps keywords are CREATE, TABLE, and REFERENCES